test/distribution/test_distribution_inference.py

```diff
@@ -808,6 +810,46 @@ def test_binned_likelihood(
     ), f"bin_prob did not match: bin_prob = {bin_prob}, bin_prob_hat = {bin_prob_hat}"
 
 
+@pytest.mark.parametrize("num_cats", [6])
+@pytest.mark.parametrize(
+    "cat_probs", [np.array([0.3, 0.1, 0.05, 0.2, 0.1, 0.25])]
+)
+@pytest.mark.parametrize("hybridize", [False])
```

Comment:
is there anything wrong with `hybridize=True`?

Reply:
I copied this part over from the binned distribution. Just tested this with hybridize=True and it seems to work.